orion/analysis.py

```diff
@@ -42,7 +42,7 @@ def analyze(pipeline, train, test=None, hyperparams=None):
     if test is None:
         test = train
 
-    pipeline = _load_pipeline(pipeline, hyperparams=None)
+    pipeline = _load_pipeline(pipeline, hyperparams)
```

Comment:
Just curious: why this change?

Reply:
to not override hyperparams sent to `analyze`

Reply:
Oh, sorry. Of course. In my head I misread this as a method definition, as if you where making mandatory an argument that before was optional :sweat_smile: